src/index.html

```diff
@@ -2,15 +2,21 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
These should not need to be added here. We use bootstrap and fontawesome at other points in the project. Those should be used instead.